Dockerfile

```diff
@@ -0,0 +1,25 @@
+# custom cinder-volume container - having python-3parclient
+FROM registry.redhat.io/rhosp-rhel8/openstack-cinder-volume:16.1
```

Comment:
Does the registry change for this?

Reply:
Yes. i verified from RH page. Around 8 days back, cinder-volume 16.1 was created.